config/application.rb

```diff
@@ -19,6 +19,10 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+# Load .env* variables before the config (Settings) initializer is
+# being run.
+Dotenv::Railtie.load
```

Comment:
```suggestion
Dotenv::Railtie.load unless Rails.env.production?
```

Reply:
Indeed, it is required as prod does not use/have the dotenv pkg.